packages/web-components/src/calendar/calendar.ts

```diff
@@ -229,10 +224,15 @@ export class Calendar extends FASTCalendar {
         if (this.secondaryPanelCells) {
           this.secondaryPanelCells.forEach(cell => cell.setAttribute('tabindex', '-1'));
           const focus = this.secondaryPanelCells[0] as HTMLElement;
-          focus.tabIndex = 6;
+          focus.tabIndex = 0;
         }
       });
     }
+
+    //Emits an event when the selected dates attribute is updated
+    if (name === 'selected-dates') {
+      this.$emit('selectedDatesChanged');
```

Comment:
Do we want to emit more information here? like the newly selected date? I'm not sure..